CLI: Source eval prompts from ai setup via EVAL_SETUP_PROMPT (#34602)
storybooks/storybook: scripts/eval/README.md

## Prerequisites

> - `**gh` CLI\*\* — installed and authenticated (`gh auth login`)

Reviewer: ```suggestion
- **`gh` CLI** — installed and authenticated (`gh auth login`)
```

The system forms a cycle:

> 1. `**sync-baselines.ts`\*\* pushes a canonical `.storybook` config to each benchmark repo so every trial starts from the same known-good baseline.

Reviewer: ```suggestion
1. `**sync-baselines.ts**` pushes a canonical `.storybook` config to each benchmark repo so every trial starts from the same known-good baseline.
```

## Prompts

The eval mirrors the real user flow exactly:

1. A real user copies the "Set up Storybook with AI" prompt from the Storybook UI — a one-line nudge (`AI_SETUP_PROMPT`) that just says _"Run `npx storybook ai setup` and follow its instructions precisely."_
2. The user pastes that into their AI agent.
3. The **agent** runs `npx storybook ai setup` itself as a tool call.
4. The agent reads the resulting project-aware markdown and follows it.

> The harness hands step (1) to the trial agent as its task. It never spawns `ai setup` itself — that's the agent's job, just like with real users.

Reviewer: ```suggestion
The harness hands steps (1) and (2) to the trial agent as its task. Eval starts at step (3).
```
